arrow-core-data/src/main/kotlin/arrow/core/computations/const.kt

```diff
@@ -0,0 +1,39 @@
+package arrow.core.computations
+
+import arrow.continuations.Effect
+import arrow.core.Const
+import arrow.core.const
+import kotlin.coroutines.RestrictsSuspension
+
+@Deprecated(
+  "Const binding does not require suspension and this computation block will be removed."
+)
+fun interface ConstEffect<A, T> : Effect<Const<A, T>> {
+
+  @Deprecated("The monadic operator for the Arrow 1.x series will become invoke in 0.13", ReplaceWith("()"))
+  suspend fun <B> Const<A, B>.bind(): B = this()
+
+  @Deprecated("The monadic operator for the Arrow 1.x series will become invoke in 0.13", ReplaceWith("()"))
+  suspend operator fun <B> Const<A, B>.not(): B = this()
+
+  suspend operator fun <B> Const<A, B>.invoke(): B =
```

Comment:
Not to open a discussion again, but something I'm not a big fan of is how invoke looks when working with the usual computations that return a monad: 

```kotlin
either {
  val speaker = service.loadSpeaker()() // or service.loadSpeaker().invoke()
  val talks = service.loadTalks(speaker.talkIds)()
}
```

This is likely how programs are gonna look overall unless I'm missing something. You end up having the double parenthesis everywhere. Also, just for relevance: On all courses we show bind as `!` or explicit `.bind()`, sometimes we also mentioned the style using destructuring. Every time they have picked the explicit one: ".bind()". The not operator is the one less embraced overall.

Reply:
Hi Jorge, thanks!! :raised_hands: Discussions have been enabled in `arrow` repository in order not to miss these things: https://github.com/arrow-kt/arrow/discussions

Reply:
Invoke was chosen because it offers us both options whereas bind is not an operator so the code is all calls to .bind() With `invoke` the user can choose an explicit name or operator. I prefer user invoke but if we want to change it we are still in time before 0.12.x

Reply:
https://kotlinlang.slack.com/archives/C5UPMM0A0/p1608320480156200